input/fsh/models.fsh

```diff
@@ -1,13 +1,13 @@
 Invariant: sql-name
 Description: """
 Name is limited to letters, numbers, or underscores and cannot start with an
-underscore -- i.e. with a regular expression of: ^[^_][A-Za-z][A-Za-z0-9_]+$ 
+underscore -- i.e. with a regular expression of: ^[A-Za-z][A-Za-z0-9_]+$ 
```

Comment:
Why are we requiring a minimum of two characters - is this a requirement in one of the database engines?